generators/app/prompts.js

```diff
@@ -190,7 +190,7 @@ function askForIncrementalOperations(context, done) {
             type: 'list',
             name: 'currentEntity',
             message: 'For which entity (class name)?',
-            choices: [...getConcernedEntities(previousConfiguration(context)), { name: 'None (leave incremental mode)', value: undefined }],
```

Comment:
The none choice disappeared?

Reply:
Yes because it's non sense, as if you choose it it does nothing. So if the user want to interrupt the prompts he just have to make a "Ctrl + C".

Reply:
seen together :+1: